File: help/pt_pt/exercicios_da_ficha_11/Ficha11_006/Ficha11_006.cpp
// Programador           -

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha11_006 {

    public:
        int main() {
            linha3() ;
            linha5() ;
            linha7() ;
            linha5() ;
            linha3() ;
            return 0;
        }

        //::::::::::::::::::::::::::::::::::::::::::::::::::
        //User Defined Functions Code
        //::::::::::::::::::::::::::::::::::::::::::::::::::
        void linha3(  ){
            int i  = 0 ;
            for( int i = 0; i < 3; i = i + 1 ) {
                cout << "*";
            }
            cout << "\n";
        }
        //::::::::::::::::::::::::::::::::::::::::::::::::::
        void linha5(  ){
            int i  = 0 ;
            for( int i = 0; i < 5; i = i + 1 ) {
                cout << "*";
            }
            cout << "\n";
        }
        //::::::::::::::::::::::::::::::::::::::::::::::::::
        void linha7(  ){
            int i  = 0 ;
            for( int i = 0; i < 7; i = i + 1 ) {
                cout << "*";
            }
            cout << "\n";
        }
};// end of class Ficha11_006
// Execute main function
int main(){
    Ficha11_006 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:09

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_006/Ficha11_006.cs
// Programador           -

using System.IO;
using System;
public class Ficha11_006 {

    //Main Function
    static void Main() {
        linha3() ;
        linha5() ;
        linha7() ;
        linha5() ;
        linha3() ;
    }

    //::::::::::::::::::::::::::::::::::::::::::::::::::
    //User Defined Functions Code
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha3( ){
        for( int i = 0; i < 3; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha5( ){
        for( int i = 0; i < 5; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }
    //::::::::::::::::::::::::::::::::::::::::::::::::::
    public static void linha7( ){
        for( int i = 0; i < 7; i = i + 1 ) {
            Console.Write("*" );
        }
        Console.Write("\n" );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:09
